docs/faq.md

```diff
@@ -36,3 +36,40 @@ We list some common troubles faced by many users and their corresponding solutio
     1. If you are using miniconda rather than anaconda, check whether Cython is installed as indicated in [#3379](https://github.com/open-mmlab/mmdetection/issues/3379).
     You need to manually install Cython first and then run command `pip install -r requirements.txt`.
     2. You may also need to check the compatibility between the `setuptools`, `Cython`, and `PyTorch` in your environment.
+
+- "Segmentation fault".
+    1. Check you GCC version and use GCC 5.4. This usually caused by the incompatibility between PyTorch and the environment (e.g., GCC < 4.9 for PyTorch). We also recommand the users to avoid using GCC 5.5 because many feedbacks report that GCC 5.5 will cause "segmentation fault" and simply changing it to GCC 5.4 could solve the problem.
+
+    2. Check whether PyTorch is correctly installed and could use CUDA op, e.g. run python and type the following
+
+        ```python
+        import torch
+        a = torch.tensor(1)
+        b = a.cuda()
+        c = b * b
+        ```
+
+        And see whether they could correctly output results.
+
+    3. If Pytorch is correctly installed, check whether MMCV is correctly installed.
+
+        ```python
+        import mmcv
+        import mmcv.ops
+        ```
+
+        If MMCV is correctly installed, then there will be no issue of the above two commands.
+
+    4. If MMCV and Pytorch is correctly installed, you may add `print` in the mmdetection code, and see which part leads the segmentation fault.
```

Comment:
You man use [ipdb](https://pypi.org/project/ipdb/) to set breakpoints or directly add 'pint' in mmdetection code...